app/components/avo/fields/common/single_file_viewer_component.html.erb

```diff
@@ -33,7 +33,7 @@
         <% end %>
       </div>
       <div>
-        <% if @resource.authorization.authorize_action(:delete_attachments?, raise_exception: false) %>
+        <% if @file.id.present? && @resource.authorization.authorize_action(:delete_attachments?, raise_exception: false) %>
```

Comment:
Doesn't the Rails model have something like `persisted?`?

Reply:
yes! thanks for remembering me, that was my first thought actually, don't remember how it ends like this ='))